### ISSUE TITLE
Activate DAO from settings

### ISSUE THREAD

**Issue by ExPrgrmmr** [bisq-network/bisq#3546]
DAO is excellent but its advanced featureres can distract new users, it also carries performance issues when syncing.

It is possible to deactivate  DAO with the '--daoActivated' command line argument right now.

For convenience DAO should be activable/deactivable from settings. DAO should be deactivated by default on an installation.

https://bisq.community/t/minimum-bandwidth-to-run-bisq/7737

**Comment by ExPrgrmmr** [comment]
Do you have to vote on this?
#bisqgrowth

**Comment by sqrrm** [comment]
The DAO is integrated with Bisq, as software and concept. For example, you need it to know the trade fees. At this point it's not reasonable to split it out, and I don't think it's a good idea either since you need it to be able to trade.

**Comment by ExPrgrmmr** [comment]
> You need it to know the trade fees. At this you need it to be able to trade.

Not true. Everything works with it disabled from the command line and one got all information. If one wants to step into BSQ or further just activate it. As a matter of fact it is a pain syncing all time locking up the gui (not responding) and drains the ram.

**Comment by ExPrgrmmr** [comment]
Some user reports here
https://bisq.community/t/bisq-version-2-implications/8638/9

**Comment by Bisq2People** [comment]
In short, while DAO is necessary for management and development it implies centralization on the Bisq application. Having it as a default choice to users will be negative for exchange liquidity.
